### PR TITLE
PRO-336 Add Optimism Network

### PR DESCRIPTION
**Pull request by hemadri-tank-bacancy**
Add new optimism network

https://user-images.githubusercontent.com/78720269/180791594-135dc047-be84-4237-9828-c32acf90519d.mp4

### CODE REVIEW

File: src/utils/uiConfig.js [hunk 1]
```diff
@@ -108,9 +117,9 @@ export function useChainConfig(chain: Chain): ChainConfig {
 
 type AssetCategoryConfig = {|
   title: string,
-  titleShort: string,
-  iconName: IconName,
-  chartColor: string,
+    titleShort: string,
+      iconName: IconName,
+        chartColor: string,
```
Comment (IAmKio) [inline src/utils/uiConfig.js:122]: And the same as above, this might just be a pattern i've never seen before...

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a few minor things but other than that - LGTM!

File: src/utils/chains.js [hunk 2]
```diff
@@ -42,6 +42,8 @@ export const chainFromChainId: { [number]: Chain } = {
   [CHAIN_ID.MUMBAI]: CHAIN.POLYGON,
   [CHAIN_ID.AVALANCHE]: CHAIN.AVALANCHE,
   [CHAIN_ID.FUJI]: CHAIN.AVALANCHE,
+  [CHAIN_ID.OPTIMISM]: CHAIN.OPTIMISM,
+  [CHAIN_ID.OPTIMISM_KOVAN]: CHAIN.OPTIMISM,
```
[the remark this answers is not in the document]
Comment (hemadri-tank-bacancy) [reply src/utils/chains.js:46]: @IAmKio Do you mean OPTIMISM_KOVAN string replace with other string?

File: src/services/etherspot.js [hunk 3]
```diff
@@ -177,6 +181,7 @@ export class EtherspotService {
   }
 
   getSdkForChain(chain: Chain): ?EtherspotSdk {
+    console.log('');
```
[the remark this answers is not in the document]
Comment (hemadri-tank-bacancy) [reply src/services/etherspot.js:184]: Resolved!

File: src/utils/chains.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (IAmKio) [reply src/utils/chains.js:46]: @hemadri-tank-bacancy sorry, i'm referring to the `CHAIN.OPTOMISM` constant. If i understand this correctly, this means that in the UI the name will resolve to "Optimism"? If it does, could we please change this to "Optimism Kovan" for clarity when running in testnet mode?

[the remark this answers is not in the document]
Comment (hemadri-tank-bacancy) [reply src/utils/chains.js:46]: "Optimism Kovan" name visible in staging

[the remark this answers is not in the document]
Comment (IAmKio) [reply src/utils/chains.js:46]: Perfect thank you

**Review by IAmKio — APPROVED** [review]
LGTM - thank you!

**Pull request merged**